### PR TITLE
Added tenacity python package

### PR DESCRIPTION
**Pull request by paulgibert**
This is the tenacity python package:  https://github.com/jd/tenacity
This would be my first contribution to wolfi or any open source project ever!
\#first-open-source-contribution

#### For new package PRs only
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`) - No `endoflife.date` found

### CODE REVIEW

File: py3-tenacity.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      expected-commit: 41ed2420cda8ab7650a39900451099f4730266c3
+
+  - uses: python/build-wheel
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: jd/tenacity
```
Comment (imjasonh) [inline py3-tenacity.yaml:28]:
```suggestion
    identifier: jd/tenacity
    use-tag: true
```

File: py3-tenacity.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    use-tag: true
```
Comment (imjasonh) [inline py3-tenacity.yaml:29]:
```suggestion
    use-tag: true

```

Comment (imjasonh) [inline py3-tenacity.yaml:29]:
```suggestion
    use-tag: true
  ignore-regex-patterns:
    - 'mergify-'
```

File: py3-tenacity.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  ignore-regex-patterns:
+    - 'mergify-'
```
Comment (imjasonh) [inline py3-tenacity.yaml:31]:
```suggestion
    - 'mergify-*'
```

Let's try this!

File: py3-tenacity.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: jd/tenacity
+    use-tag: true
+  ignore-regex-patterns:
+    - 'mergify-*'
```
Comment (imjasonh) [inline py3-tenacity.yaml:31]:
```suggestion
    - 'mergify2-'
```

**Comment by imjasonh** [comment]
MSqziH

**Review by imjasonh — APPROVED** [review]

**Comment by jspeed-meyers** [comment]
Thank you, @imjasonh!

And nice work, @paulgibert!

**Pull request merged**